src/App.js

```diff
@@ -1,33 +1,14 @@
 import './App.css';
-import PropTypes from 'prop-types';
-import { createStore } from 'redux';
-import { Provider } from 'react-redux';
-import mainReducer from './reducers/index';
 import BooksList from './containers/BookList';
-import BookForm from './containers/BookForm';
+import BookForm from './components/BookForm';
 
 function App(/* props */) {
-  const store = createStore(mainReducer);
-  const { books } = store.getState();
   return (
     <div className="App">
-      <Provider store={store}>
-        <BookForm />
-        <BooksList books={books} />
-      </Provider>
+      <BookForm />
+      <BooksList />
```

Comment:
- [ ] Since we are following the approach to separate components into Presentational and Container components, it makes sense to move `App.js` component to the `components/` directory.